### PR TITLE
Notes for November 2022 JupyterHub team meeting

### PR DESCRIPTION
**Pull request by manics**
Closes https://github.com/jupyterhub/team-compass/issues/576

### CODE REVIEW

File: docs/meetings/community/2022-11-15.md [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+- name / institution / GitHub handle
+- Christian Dike/ - /@christiandike
+- Saka Abass / Outreachy / Olaedey
+- Min RK / Simula / @minrk
+- Arafat Abdussalam / Outreachy / ArafatAbdussalam
+- Simon Li / University of Dundee / @manics
+- Sarah Gibson / 2i2c / sgibson91
+- Sylvain Corlay / QuantStack / SylvainCorlay / sylvain.corlay@quantstack.net
+- A T Darian / QuantStack / @afshin / afshin.darian@quantstack.net
+- Gabriela Vives / Quantstack / GabrielaVives / gabriela.vives@quantstack.net
```
Comment (sgibson91) [inline docs/meetings/community/2022-11-15.md:32]: I can't seem to add a suggestion 😢 But let's remove the emails from the official record? I'll make sure to pass them onto the 2i2c folk

Comment (choldgraf) [reply docs/meetings/community/2022-11-15.md:32]: +1 at least so that we don't we don't cause bot-spam for people

Comment (manics) [reply docs/meetings/community/2022-11-15.md]: I've forced pushed them away. Do we need to make it clearer in the template that people shouldn't add their emails? Or should people be free to add them if they want to be contacted?

Comment (sgibson91) [reply docs/meetings/community/2022-11-15.md]: I think this is a one-off. I explicitly asked them to add some contact details so 2i2c could reach out regarding collaborating on UI/UX research regarding Binder launching, and adding them to the hackmd was the quickest way to transfer that info in the moment.

File: docs/meetings/community/2022-11-15.md [hunk 2]
```diff
@@ -0,0 +1,113 @@
+---
+tags: binder, meeting, notes
+---
+
+# JupyterHub and BinderHub Team Meeting
+
+- **Date:** Tuesday 15th November 2022
+- **Time:** 6PM UTC
```
Comment (sgaist) [inline docs/meetings/community/2022-11-15.md:8]: Just one note: it seems there has been something wonky happening with the meeting time maybe related to DST.

I thought it was at 7PM CET for me yesterday (UTC+2 with DST) but when I checked the hackmd, it was noted as 5PM UTC so 6 for me (following the arewemeetingyet link and I forgot that we reverted to UTC+1). However when I connected, there was another fine gentleman alone there and we were both wondering whether we had missed something.

I then checked the google calendar on [the meeting page on readthedoc](https://jupyterhub-team-compass.readthedocs.io/en/latest/meetings/index.html) which showed the meeting being at 10AM.

I wanted to check the October meeting notes to see if I had missed something but I couldn't find them. Now that I have rechecked the google calendar, it seems that is shows the time for North America, hence the confusion with regard to that part.

**Pull request merged**